### PR TITLE
Update DomA start conditions

### PR DESCRIPTION
**Pull request by arminn**

### CODE REVIEW

File: meta-xt-prod-cockpit-rcar-control/recipes-guests/doma/doma.bbappend [hunk 1]
```diff
@@ -6,11 +6,13 @@ SRC_URI += "\
     file://doma-vdevices.cfg \
     file://doma-set-root \
     file://doma-set-root.conf \
+    file://doma.service \
```
[the remark this answers is not in the document]
Comment (arminn) [reply meta-xt-prod-cockpit-rcar-control/recipes-guests/doma/doma.bbappend:9]: Personally, i don't what to relay on yocto "magic" like "path will be first in the search list", and keep change simple. The /etc is the official way to override the whole unit file, so I prefer to keep it that way.

**Comment by arminn** [comment]
@rshym  your comments adressed

**Comment by rshym** [comment]
Reviewed-by: Ruslan Shymkevych <<ruslan_shymkevych@epam.com>>

**Comment by arminn** [comment]
rb added

**Pull request merged**